src/styles/main.scss

```diff
@@ -1,3 +1,50 @@
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.antisnake {
+  display: grid;
+  grid-template-columns: repeat(3, minmax(300px, 1fr));
+}
+
+.antisnake__item {
+  min-width: 300px;
+  text-align: center;
+  line-height: 300px;
+  font-size: 100px;
+  color: white;
```

Comment:
change color to the decimal code, also it better when each color have a variable